woocommerce-payments.php

```diff
@@ -8,10 +8,10 @@
  * Woo: 5278104:bf3cf30871604e15eec560c962593c1f
  * Text Domain: woocommerce-payments
  * Domain Path: /languages
- * WC requires at least: 6.6
- * WC tested up to: 6.9.0
+ * WC requires at least: 6.8
+ * WC tested up to: 7.0.0
  * Requires at least: 5.8
- * Requires PHP: 7.0
+ * Requires PHP: 7.2
```

Comment:
There isn't discussion, but I bumped it because of the latest supported PHP version in WooCommerce. I can revert it if you think that this is mistake.

Reply:
I don't think it's a mistake @zmaglica, but this would force a merchant to upgrade PHP version to `7.2` if they're currently running `7.0` and might cause some problems if we bump this without prior notice.

Right now, anyone running PHP 7.0 or above can run WC Payments. So let's have a discussion before bumping this one. WDYT?

Reply:
Make sense. I will revert PHP version change and add it to the retro post.